Web/starter_project_vanguard/components/layout/links/socialMediaLinks.tsx

```diff
@@ -0,0 +1,31 @@
+import { FaFacebook, FaInstagram, FaLinkedin,FaTwitter,FaYoutube} from 'react-icons/fa'
+
+interface iconLink {
+    icon: any
+    url : string
+  }
+  
+  export const socialMedialinks : iconLink[] = [
+
+    {
+        icon: <FaTwitter />,
```

Comment:
using components in a list is fine when having a list inside another component, but if it were defined inside the data folder, it would required a workaround